src/wwwroot/styles/views/_settings.scss

```diff
@@ -620,6 +620,67 @@
   }
 }
 
+.plan--box-container {
+  .arrow--box-up {
+    position: relative;
+    background: $suit-font-color-bright;
+    border: 1px solid $suit-accent-color;
+    color: $suit-main-color;
+    top: 10px;
+    padding: 35px;
+    &:after{
+      bottom: 100%;
+      left: 24%;
+      border: solid transparent;
+      content: " ";
+      position: absolute;
+      border-bottom-color: $suit-font-color-bright;
+      border-width: 8px;
+      margin-left: -8px;
+    }
+    &:before{
+      bottom: 100%;
+      left: 24%;
+      border: solid transparent;
+      content: " ";
+      position: absolute;
+      border-bottom-color: $suit-accent-color;
+      border-width: 9px;
+      margin-left: -9px;
+    }
+    ol{
+      padding-left: 15px;
+      margin-bottom: 0px;
+      li{
+        border-left: 0px;
```

Comment:
those borders :(, if it has to be like this, then leave it, we can fix that later.

Reply:
I attached an image to explain why I need to override this value explicitly.
![image](https://user-images.githubusercontent.com/7245667/38092294-38c23ac4-333e-11e8-9b19-ce81bd7b8fe5.png)